wp-includes/widgets/class-wp-widget-media.php

```diff
@@ -147,7 +147,7 @@ public function is_attachment_with_mime_type( $attachment, $mime_type ) {
 		if ( 'attachment' !== $attachment->post_type ) {
 			return false;
 		}
-		return strtok( $attachment->post_mime_type, '/' ) === $mime_type;
+		return wp_attachment_is( $mime_type, $attachment ) || strtok( $attachment->post_mime_type, '/' ) === $mime_type;
```

Comment:
I can't think of a situation right now. Fine to restrict to `wp_attachment_is()` for now.